.github/workflows/on-push-to-main-branch.yml

```diff
@@ -204,6 +204,33 @@ jobs:
           template_file: ./examples/nodejs/aws/secrets-manager/README.template.md
           output_file: ./examples/nodejs/aws/secrets-manager/README.md
 
+      - name: Generate README for examples/nodejs/aws/eventbridge
```

Comment:
you need to do this in the PR workflow file too

Reply:
I forgot, will do. Thanks!